README.md

```diff
@@ -6,7 +6,11 @@ Contracts of iExec Voucher project.
 
 ### API Documentation
 
-The the API documentation can be found in [docs/](./docs/index.md).
+The API documentation can be found in [docs/](./docs/index.md).
+
+### UML diagram
+
+UML class diagram for voucher contracts can be found [here](./docs/class-diagram.svg). From there open it with "Raw" or "Download"  (upper right corner).
```

Comment:
```suggestion
UML class diagram for voucher contracts can be found [here](./docs/class-diagram.svg).
Raw version is available [here](./docs/class-diagram.svg?raw=true) for better readability.

Reply:
Ah very nice I was looking for this thanks!